x-pack/plugins/ingest_manager/server/services/agent_policy.ts

```diff
@@ -260,6 +261,19 @@ class AgentPolicyService {
   ): Promise<AgentPolicy> {
     return this._update(soClient, id, {}, options?.user);
   }
+  public async bumpAllAgentPolicies(
+    soClient: SavedObjectsClientContract
+  ): Promise<Promise<SavedObjectsBulkUpdateResponse<AgentPolicy>>> {
+    const currentPolicies = await soClient.find<AgentPolicySOAttributes>({
```

Comment:
I decided to use find() here though it could take longer than a bulkGet, but saves me from having to fetch for the ids.